cmd/sovereignnode/config/common.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import (
+	"github.com/multiversx/mx-chain-core-go/core"
+)
+
+// LoadSovereignNotifierConfig returns NotifierConfig by reading it from the provided file
+func LoadSovereignNotifierConfig(filepath string) (*NotifierConfig, error) {
+	cfg := &NotifierConfig{}
+	err := core.LoadTomlFile(cfg, filepath)
+	if err != nil {
+		return nil, err
+	}
+
+	return cfg, err
```

Comment:
return cfg, nil or return core.LoadTomlFile(cfg, filepath)  and remove L11-L15

Reply:
+1

Reply:
Right, fixed return here.

Cannot replace L11-15, need to load the `cfg` into `&NotifierConfig` object + `LoadTomlFile` only returns error